deployer/templates/es.yaml

```diff
@@ -92,7 +96,7 @@ objects:
                   mountPath: /etc/elasticsearch/secret
                   readOnly: true
                 - name: elasticsearch-config
-                  mountPath: /usr/share/elasticsearch/config
+                  mountPath: /etc/elasticsearch/config
```

Comment:
I ended up doing a bunch of odd things to appease the java security manager, I believe that ES picks up changes from the configmap currently... but now I'm not sure how :-/ I'll probably need to rip and replace some things around the configs, this included.